lang/cs/BUILD.md

```diff
@@ -24,7 +24,13 @@ Prerequisites
 -------------
 
   * Windows OS.
-  * [Building REEF Java](../java/BUILD.md).
+  * Apache maven
```

Comment:
@tcNickolas I went ahead and removed the duplication here, they now only exist in the java file, once my newest pull request comes out this change will be seen